### PR TITLE
Debug mode: additional logging

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added additional logging in debug mode about the docker image and integration brand+instance. Example:

```
2020-11-01T12:20:59.661221+02:00 info: (Generic_SQL_postgres8769e4dc-3a4c-4f86-8b88-d259268f258f_Generic SQL_test-module) debug-mode started.
http client print found: False.
#### Env {'PATH': '/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', 'HOSTNAME': 'e222fb502620', 'HTTP_PROXY': '', 'http_proxy': '', 'HTTPS_PROXY': '', 'https_proxy': '', 'LANG': 'C.UTF-8', 'GPG_KEY': 'E3FF2839C048B25C084DEBE9B26995E310250568', 'PYTHON_VERSION': '3.8.5', 'PYTHON_PIP_VERSION': '20.2.2', 'PYTHON_GET_PIP_URL': 'https://github.com/pypa/get-pip/raw/5578af97f8b2b466f4cdbebe18a3ba2d48ad1434/get-pip.py', 'PYTHON_GET_PIP_SHA256': 'd4d62a0850fe0c2e6325b2cc20d818c580563de5a2038f917e3cb0e25280b4d1', 'DOCKER_IMAGE': 'demisto/genericsql:1.1.0.11281', 'HOME': '/'}.
#### Params: {'connect_parameters': None, 'credentials': {'credential': '', 'credentials': {'id': '', 'locked': False, 'modified': '0001-01-01T00:00:00Z', 'name': '', 'password': '', 'sortValues': None, 'sshkey': '', 'sshkeyPass': '', 'user': '', 'vaultInstanceId': '', 'version': 0, 'workgroup': ''}, 'identifier': 'postgres1', 'password': '<XX_REPLACED>', 'passwordChanged': False}, 'dbname': 'postgres', 'dialect': 'PostgreSQL', 'host': 'myhost.test.com', 'pool_ttl': '600', 'port': '5432', 'ssl_connect': False, 'use_pool': False}.
#### Docker image: [demisto/genericsql:1.1.0.11281]
#### Integration: brand: [Generic SQL] instance: [Generic_SQL_postgres8769e4dc-3a4c-4f86-8b88-d259268f258f]

```

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -3443,10 +3443,15 @@ def log_start_debug(self):
         """
         Utility function to log start of debug mode logging
         """
-        msg = "debug-mode started.\nhttp client print found: {}.\nEnv {}.".format(self.http_client_print is not None,
-                                                                                  os.environ)
+        msg = "debug-mode started.\nhttp client print found: {}.\n#### Env {}.".format(self.http_client_print is not None,
+                                                                                       os.environ)
         if hasattr(demisto, 'params'):
-            msg += "\nParams: {}.".format(demisto.params())
+            msg += "\n#### Params: {}.".format(demisto.params())
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:3449]: lets print dictionaries with json.dumps with indent=4

**Review by anara123 — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (glicht) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:3449]: sounds good. will add

**Comment by glicht** [comment]
@ShirleyDenkberg @kirbles19 waiting for your review

**Comment by kirbles19** [comment]
@glicht Done

**Pull request merged**